launch/start_debug.launch

```diff
@@ -1,63 +0,0 @@
-<?xml version="1.0"?>
-<launch>
```

Comment:
@MaxBaeten , @MatthijsBurgh do you ever use this? Otherwise this launch file will be removed. If you want to keep it, move it to another directory so that no alias etc. is created.